qtypes/_units.py

```diff
@@ -112,47 +73,17 @@ def converter(val, current_unit, destination_unit):
 convert = converter
 
 
-def get_symbol(units) -> Optional[str]:
-    """Get default symbol type.
-
-    Parameters
-    ----------
-    units_str : string
-        Units.
-
-    Returns
-    -------
-    string
-        LaTeX formatted symbol.
-    """
-    quantity = ureg.Quantity(1, ureg[units])
-    if quantity.check("[length]"):
-        return r"\lambda"
-    elif quantity.check("1 / [length]"):
-        return r"\bar\nu"
-    elif quantity.check("[energy]"):
-        return r"\hslash\omega"
-    elif quantity.check("1 / [time]"):
-        return "f"
-    elif quantity.check("[time]"):
-        return r"\tau"
-    elif quantity.check("[fluence]"):
-        return r"\mathcal{F}"
-    elif quantity.check("[temperature]"):
-        return "T"
-    elif ureg[units] in (ureg.deg, ureg.radian):
-        return r"\omega"
-    else:
-        return None
-
-
 def get_valid_conversions(units, options=blessed_units) -> tuple:
+    if units in options:
+        return options[units]
     return tuple(i for i in options if is_valid_conversion(units, i))
 
 
 def is_valid_conversion(a, b, blessed=True) -> bool:
     if a is None:
         return b is None
+    if a in blessed_units:
+        return b in blessed_units[a]
```

Comment:
should only apply if `blessed` is true, I think... and pay attention to the next if condition as well...